### PR TITLE
Reorganize documentation for GitHub pages

### LINKED ISSUE #371: Move documentation to GitHub Wiki
The documentation is fairly thorough and complete but it seems as though it's starting to become somewhat unorganized; the Markdown files are being written and tossed into `docs/` with links in `README.md` but perhaps breaking things down into sensible chunks of information and organizing them in the repo's Wiki would make things more easily navigable.

### PR DESCRIPTION
**Pull request by mmattioli**
Reorganize documentation for clarity and use with GitHub pages static
site generator. This fixes #371.

### CODE REVIEW

File: docs/server-centos6.md [hunk 1]
```diff
@@ -1,42 +1,48 @@
-# Algo pre-install steps for Red Hat/CentOS 6.x (currently 6.8)
+# RedHat/CentOS 6.x server setup
```
Comment (dguido) [inline docs/server-centos6.md]: I would still call this "RedHat/CentOS 6.x pre-install steps." This is only intended to run the Algo ansible scripts, not host a server.

File: docs/client-generic.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Client installation
+# Generic client setup
```
Comment (dguido) [inline docs/client-generic.md]: I think this is a Linux client setup. There are two different kinds of Linux clients:
* Linux servers (w/ raw strongSwan)
* Linux desktops (w/ Network Manager)

Let's call this client-linux.md and "Linux client setup" and I'll call the other one client-networkmanager.md and "Network Manager client setup"?

File: docs/client-windows.md [hunk 3]
```diff
@@ -1,6 +1,4 @@
-# Windows
-
-## How to run Algo on Windows 10
+# Windows client setup
 
 Before run Algo, you have to have:
```
Comment (dguido) [inline docs/client-windows.md]: oops, can you change this line to say "Prerequisites:"?

**Comment by dguido** [comment]
Great PR! Thanks! I left a few minor comments. Happy to accept this with those minor changes.

**Comment by mmattioli** [comment]
@dguido, I just updated with the requested changes.

**Comment by dguido** [comment]
Thanks!

**Pull request merged**